frontend/views/components/LinkToCopy.vue

```diff
@@ -47,14 +47,25 @@ export default ({
     }
   },
   methods: {
+    // FIXME: probably needs to be defined centrally somewhere else
+    isMacSafari (userAgent) {
+      const safari = /Macintosh.*Safari/
+      const chrome = /Macintosh.*Chrome/
+      const edge = /Macintosh.*Edg/
+      return userAgent.match(safari) &&
+              !(userAgent.match(chrome) || userAgent.match(edge))
+    },
     copyToClipboard () {
       // if the user is using the device that supports web share API, use it and then skip the other logics below.
       if (navigator.share) {
         navigator.share({
           title: this.L('Your invite'),
           url: this.link
-        })
-        return
+        }).catch((error) => console.log(error))
+        // we return immediately for any case other than mac safari
+        if (!this.isMacSafari(navigator.userAgent)) {
+          return
+        }
```

Comment:
Wouldn't it make more sense to put this into the condition above, like so?

```js
if (navigator.share && !this.isMacSafari()) {
```

Reply:
i was under the impression that for Mac Safari (which we assume would have `navigator.share` defined), we would still want to display the share context menu AND copy the link to the clipboard (only because the copy to clipboard is missing in the share context menu)
which would sort of equal to other browsers ie Edge and Chrome (Windows) which have `navigator.share` defined AND has the option to copy the link to the clipboard, amirite? :)

but admittedly copying the url to clipboard might not be the nest UX experience ("were doing something that the user didnt choose to do")

Reply:
So, the issue was to disable the share the share context menu on macOS Safari entirely, right? Since it doesn't really work how we want it to.

We want it to behave exactly like Firefox on macOS behaves - simply displaying the "copied to clipboard" tooltip

Reply:
ok

Reply:
changed